### PR TITLE
UI-1344 fixes for better lighthouse scores

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UI-1344

### CODE REVIEW

File: src/vue/backend/src/components/listingsCard.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
                 v-bind="listing"
             >
             <template v-slot:propertyCardImage>
-                    <img class="property-card__image" :src="listing.image">
+                    <img class="property-card__image" :src="listing.image" alt="listing image">
```
Comment (sgreenl5) [inline src/vue/backend/src/components/listingsCard.vue:14]: For listing image, we can have it leverage the listings address. Something like
`:alt="`Image of ${listing.address}`"
that way the alt tag describes the image better

File: src/vue/backend/src/templates/AgentsSettings.vue [hunk 2]
```diff
@@ -27,7 +27,7 @@
             <p>Default Number of Posts Description of the Default Number of Posts setting in IMPress Agents. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
         </idx-block>
         <idx-form-group>
-            <idx-form-label customClass="form-content__label" for="number-of-posts">Default Number of Posts</idx-form-label>
+            <idx-form-label customClass="form-content__label" target="number-of-posts">Default Number of Posts</idx-form-label>
```
Comment (sgreenl5) [inline src/vue/backend/src/templates/AgentsSettings.vue:30]: The targets and labels will need to be prefixed with the `idxStrap` prefix. Same for all target/id pairs

**Review by sgreenl5 — CHANGES_REQUESTED** [review]

File: src/vue/backend/src/templates/AgentsSettings.vue (hunk 2 above)
Comment (nicmarson) [reply src/vue/backend/src/templates/AgentsSettings.vue:30]: The `id` isn't being prefixed with `idx-wp-` so leaving the `idxStrap` prefix off makes them match.
Screen Shot 2021-03-24 at 7 41 09 AM

Comment (nicmarson) [reply src/vue/backend/src/templates/AgentsSettings.vue:30]: I think I get what you're saying here, I should add the prefix to both the label and its target...

File: src/vue/backend/src/templates/GeneralSettings.vue [hunk 3]
```diff
@@ -6,14 +6,15 @@
...
         <idx-form-group>
-            <idx-form-label customClass="form-content__label" for="website-wrapper">
-                <idx-block tag="h2" className="form-content__title">Name Your Global Website Wrapper</idx-block>
+            <idx-block className="form-content__label">
+                <idx-block id="wrapperNameLabel" tag="h2" className="form-content__title">Name Your Global Website Wrapper</idx-block>
                 <p>Wrappers set the overall styling of your IDX Broker pages, some words about maintaining a consistent design between WordPress and IDX Broker.</p>
-            </idx-form-label>
+            </idx-block>
             <idx-form-input
+                aria-labelledby="wrapperNameLabel"
```
Comment (sgreenl5) [inline src/vue/backend/src/templates/GeneralSettings.vue:14]: Missing prefixes

File: src/vue/backend/src/templates/impressListingsIdxContent.vue [hunk 4]
```diff
@@ -97,12 +97,14 @@
             </idx-block>
         </idx-form-group>
         <idx-form-group>
-            <idx-form-label customClass="form-content__label">
-                <idx-block tag="h3" className="form-content__title">Import Title</idx-block>
+            <idx-block className="form-content__label">
+                <idx-block tag="h3" id="importTitleLabel" className="form-content__title">Import Title</idx-block>
```
Comment (sgreenl5) [inline src/vue/backend/src/templates/impressListingsIdxContent.vue:101]: Missing prefixes

**Review by sgreenl5 — COMMENTED** [review]
A few missing prefixes

**Review by sgreenl5 — APPROVED** [review]

**Pull request merged**